docs/csharp/language-reference/statements/selection-statements.md

```diff
@@ -0,0 +1,107 @@
+  - "default keyword [C#]"
+---
+# Selection statements (C# reference)
+
+The following statements select statements to execute out of a number of possible statements based on the value of an expression:
+
+- The [`if` statement](#the-if-statement): selects a statement to execute based on the value of a Boolean expression.
+- The [`switch` statement](#the-switch-statement): selects a statement list to execute based on a pattern match with an expression.
+
+## The `if` statement
+
+An `if` statement can be any of the following two forms:
+
+- An `if` statement with an `else` part selects one of the two statements to execute based on the value of a Boolean expression, as the following example shows:
+
+  :::code language="csharp" interactive="try-dotnet-method" source="snippets/selection-statements/IfStatement.cs" id="IfElse":::
+
+- An `if` statement without an `else` part executes its body only if a Boolean expression evaluates to `true`, as the following example shows:
+
+  :::code language="csharp" interactive="try-dotnet-method" source="snippets/selection-statements/IfStatement.cs" id="OnlyIf":::
+
+You can nest `if` statements to check multiple conditions, as the following example shows:
+
+:::code language="csharp" interactive="try-dotnet-method" source="snippets/selection-statements/IfStatement.cs" id="NestedIf":::
+
+In an expression context, you can use the [conditional operator `?:`](../operators/conditional-operator.md) to execute one of the two expressions based on the value of a Boolean expression.
+
+## The `switch` statement
+
+The `switch` statement selects a statement list to execute based on a pattern match with a match expression, as the following example shows:
+
+:::code language="csharp" source="snippets/selection-statements/SwitchStatement.cs" id="Example":::
+
+At the preceding example, the `switch` statement uses the following patterns:
+
+- A [relational pattern](../operators/patterns.md#relational-patterns): to compare an expression result with a constant.
+- A [constant pattern](../operators/patterns.md#constant-pattern): to test if an expression result equals a constant.
+
+For information about the patterns supported by the `switch` statement, see [Patterns](../operators/patterns.md).
+
+The preceding example also demonstrates the `default` case. The `default` case specifies statements to execute when a match expression doesn't match any other case pattern. If a match expression doesn't match any case pattern and there is no `default` case, control falls through a `switch` statement.
+
+A `switch` statement evaluates the first section whose case pattern matches a match expression and whose [case guard](#case-guards), if present, evaluates to `true`. A `switch` statement evaluates case patterns in text order from top to bottom. The compiler generates an error when a `switch` statement contains an unreachable case. That is a case that is already handled by an upper case or whose pattern is impossible to match.
```

Comment:
Some slight rewording here, also italicizing the spec terms when first introduced.

 ```suggestion
A `switch` statement executes the *statement list* in the first *switch section* whose *case pattern* matches a match expression and whose [case guard](#case-guards), if present, evaluates to `true`. A `switch` statement evaluates case patterns in text order from top to bottom. The compiler generates an error when a `switch` statement contains an unreachable case. That is a case that is already handled by an upper case or whose pattern is impossible to match.
```